src/rcore.c

```diff
@@ -661,10 +662,12 @@ void InitWindow(int width, int height, const char *title)
 #if defined(SUPPORT_MODULE_RTEXT) && defined(SUPPORT_DEFAULT_FONT)
     if ((CORE.Window.flags & FLAG_WINDOW_HIGHDPI) > 0)
     {
-        // Set default font texture filter for HighDPI (blurry)
-        // RL_TEXTURE_FILTER_LINEAR - tex filter: BILINEAR, no mipmaps
-        rlTextureParameters(GetFontDefault().texture.id, RL_TEXTURE_MIN_FILTER, RL_TEXTURE_FILTER_LINEAR);
-        rlTextureParameters(GetFontDefault().texture.id, RL_TEXTURE_MAG_FILTER, RL_TEXTURE_FILTER_LINEAR);
+        #if !defined(__APPLE__)
```

Comment:
NOTE: I'm not sure if this logic for rlTextureParameters is needed when using SDL3 as it seems to render font's much blurrier when HiDPI is enabled. Definitely should be ignored on Apple systems, but possibly Windows and Linux as well. Needs more testing...